Reply/app/src/main/res/navigation/navigation_graph.xml

```diff
@@ -19,10 +19,18 @@
     <fragment
         android:id="@+id/homeFragment"
         android:name="com.materialstudies.reply.ui.home.HomeFragment"
-        android:label="HomeFragment" >
+        android:label="HomeFragment">
+        <argument
+            android:name="mailbox"
+            app:argType="com.materialstudies.reply.ui.home.Mailbox"
+            android:defaultValue="INBOX"/>
         <action
             android:id="@+id/action_homeFragment_to_emailFragment"
             app:destination="@id/emailFragment" />
+        <action
+            android:id="@+id/action_homeFragment_to_homeFragment"
+            app:destination="@+id/homeFragment"
+            app:launchSingleTop="true"/>
```

Comment:
so this means that there's only one instance of the HomeFragment right? and if you use system back after switching mailboxes, it would exit the app instead of going back to the Inbox?

Reply:
Yea, that's right.
I was debating on what the behavior should be here but I think it makes more sense having the app exit on back instead of moving back through the mailboxes you've navigated from.

Reply:
gotcha. i'm always surprised when android app's exit when going back after a navigation menu change. i was thinking the ideal behavior would be that if you are looking at any mailbox other than Inbox, then system back would take you to Inbox. meaning there's a max of 2 HomeFragment's in the stack. but that may be more complexity than it's worth.

Reply:
Yea, I agree.
I added a onBackPressedCallback to HomeFragment that makes sure Inbox is the last fragment before exiting. I don't think this adds too much complexity.